FlowCrypt/Controllers/Threads/AlertsFactory.swift

```diff
@@ -7,21 +7,24 @@
 //
 
 import UIKit
+import FlowCryptCommon
 
 enum AlertsFactory {
     typealias PassPhraseCompletion = ((String) -> Void)
     typealias CancelCompletion = (() -> Void)
 
     static func makePassPhraseAlert(
+        title: String = "setup_enter_pass_phrase".localized,
+        in view: UIView,
```

Comment:
`view` parameter can be removed, as `popoverPresentation` isn't needed for `alert` here

Reply:
please check issue in PR description #1269
Just as an example of similar issue 
https://stackoverflow.com/questions/31577140/uialertcontroller-is-crashed-ipad

Reply:
Yes, I read original issue, error there says `Your application has presented a UIAlertController (<UIAlertController: 0x7f88e2872000>) of style UIAlertControllerStyleActionSheet`

The same for stackoverflow post, it shows `.actionSheet`, not `.alert`:
`let shareMenu = UIAlertController(title: nil, message: "Share using", preferredStyle: .ActionSheet)`

So we need `popoverPresentationStyle` set only for UIAlertController with `preferredStyle: .actionSheet`

I just tested it on iPad - `preferredStyle: .alert` works well without `popoverPresentationStyle`

Reply:
strange why I have this error on .alert

Reply:
are you testing it on simulator?

Reply:
Yes, on simulator

Reply:
Looks like indeed not needed for alert style. Please check latest changes